src/server/nuclearnet/fake_nuclearnet_client.ts

```diff
@@ -36,9 +36,9 @@ export class FakeNUClearNetClient implements NUClearNetClient {
   public connect(options: NUClearNetOptions): () => void {
     this.peer = {
       name: options.name,
-      // Address and port are not used for anything yet, dummy values used.
-      address: 'fake_address',
-      port: NaN,
+      address: '127.0.0.1',
+      // TODO (Annable): Inject a random function.
+      port: Math.floor(Math.random() * (65536 - 1024) + 1024),
```

Comment:
Ephemeral ports are usually much larger than 1024... The Internet Assigned Numbers Authority (IANA) suggests the range 49152 to 65535 (2^15+2^14 to 2^16−1) for dynamic or private ports.
Bu this is probably fine.